packages/design-tokens/design-token-previews/utils/previews.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import includes from "lodash/includes";
+
+import { TokenEntry } from "../types/Token";
+
+import { FontSizePreview } from "../FontSizePreview";
+import { FontWeightPreview } from "../FontWeightPreview";
+import { IconSizePreview } from "../IconSizePreview";
+
+export const createFontWeightPreview = ([
```

Comment:
Do we need this file, or would it be better to create anonymous functions on the `constants.tsx`. e.g., 

```typescript
  [getKey(fontWeightTokens)]: [
    { name: "Pixels", transform: getValue },
    { name: "Preview", transform: ([tokenSuffix, token]: TokenEntry) => <FontWeightPreview fontWeight={token.value} key={tokenSuffix } />},
  ],
```

Reply:
To me it makes more sense to keep this file. If we put them all in the constants.ts it feels to messy to me.